CMakeGlobals.txt

```diff
@@ -23,36 +23,29 @@ set(RSTUDIO_CMAKE_GLOBALS_INCLUDED YES)
 
 # version info
 if ("$ENV{RSTUDIO_VERSION_MAJOR}" STREQUAL "")
-  string(TIMESTAMP CPACK_PACKAGE_VERSION_MAJOR "%Y-%m")
+  string(TIMESTAMP CPACK_PACKAGE_VERSION_MAJOR "%Y")
   set(RSTUDIO_UNVERSIONED_BUILD TRUE)
 else()
   set(CPACK_PACKAGE_VERSION_MAJOR $ENV{RSTUDIO_VERSION_MAJOR})
 endif()
 if ("$ENV{RSTUDIO_VERSION_MINOR}" STREQUAL "")
-  set(CPACK_PACKAGE_VERSION_MINOR "9")
+   string(TIMESTAMP CPACK_PACKAGE_VERSION_MINOR "%m")
 else()
   set(CPACK_PACKAGE_VERSION_MINOR $ENV{RSTUDIO_VERSION_MINOR})
 endif()
 if ("$ENV{RSTUDIO_VERSION_PATCH}" STREQUAL "")
-  set(CPACK_PACKAGE_VERSION_PATCH "9")
+  set(CPACK_PACKAGE_VERSION_PATCH "999")
 else()
   set(CPACK_PACKAGE_VERSION_PATCH $ENV{RSTUDIO_VERSION_PATCH})
 endif()
-set(CPACK_PACKAGE_VERSION "${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}")
+set(CPACK_PACKAGE_VERSION_SUFFIX $ENV{RSTUDIO_VERSION_SUFFIX})
```

Comment:
Should we have a default suffix if one isn't defined in the environment? (Bonus points if it helps distinguish development builds, which typically don't have env vars set, from lab builds, which do; a job formerly done by the 9/99 parts of the version)

Reply:
What do you think about `-developer+999` or something like that?

Reply:
I'm a fan. Maybe just `-dev+999` for brevity?